roles/reproducer/tasks/ci_data.yml

```diff
@@ -2,17 +2,28 @@
 - name: Run block only if we want to re-run a job
   when:
     - cifmw_job_uri is defined
+  vars:
+    regex_pattern: '{{ "^https://[\w|/].*/[a-f0-9]{7}" }}'
```

Comment:
Uhmm this variable doesn't need jinja templating so it can be perfectly fine if defined like this:
`regex_pattern: "^https://[\w|/].*/[a-f0-9]{7}"